scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -13,6 +13,7 @@ public class BuildVersion
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
         public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc2-3{CommitCountString}";
+        public string HostVersion => $"{Major}.{Minor}.1-rc2-{CommitCountString}-00";
```

Comment:
Sorry that turned out to be too convoluted.

1. All build machines produce nupkg
2. A separate definition scans azure blob storage and pushes the full list of packages.